typescript/__tests__/testProgramaticallyCreateConfig.ts

```diff
@@ -0,0 +1,20 @@
+import { AIConfigRuntime } from "../lib/config";
+import { InferenceSettings } from "../types";
+
+describe("test extract_override_settings", () => {
```

Comment:
Update description. Doesn't look correct for this test

Reply:
Updated to `test Get Global Settings` to accurately describe which method its testing